go.mod

```diff
@@ -3,6 +3,7 @@ module github.com/smartystreets/goconvey
 require (
 	github.com/gopherjs/gopherjs v0.0.0-20181017120253-0766667cb4d1 // indirect
 	github.com/jtolds/gls v4.20.0+incompatible
+	github.com/otiai10/copy v1.5.1
```

Comment:
This seems like an overkill dependency for just copying the web/client directory; I think there is probably a very short implementation for this specific use case with filepath.Walk?

Reply:
Actually... on that note, what prevents us from just using the client files from the directory they're in? The only thing that really needs to be instanced are output files, right?

Reply:
> client files from the directory they're in?
Not much, I'll double check that.

> otiai10/copy

I can replace it with 100-200 LoC for recursive copying, as a separate go file, if it's ok ?

Reply:
I guess what I was getting at was that we know the source directory doesn't have any symlinks or special files, and we know the destination directory is empty (we just created it); but of course if we can serve the files directly from the source without copying them then that's even simpler :)